adapters/beachfront/beachfront.go

```diff
@@ -4,71 +4,56 @@ import (
 	"encoding/json"
 	"errors"
 	"fmt"
+	"github.com/mxmCherry/openrtb"
+	"github.com/prebid/prebid-server/adapters"
 	"github.com/prebid/prebid-server/errortypes"
+	"github.com/prebid/prebid-server/openrtb_ext"
 	"net/http"
+	"reflect"
 	"strconv"
 	"strings"
-
-	"github.com/prebid/prebid-server/adapters"
-	"github.com/prebid/prebid-server/openrtb_ext"
-
-	"github.com/mxmCherry/openrtb"
 )
 
 const Seat = "beachfront"
 const BidCapacity = 5
 
-const BannerEndpoint = "https://display.bfmio.com/prebid_display"
-const VideoEndpoint = "https://reachms.bfmio.com/bid.json?exchange_id="
+const bannerEndpoint = "https://display.bfmio.com/prebid_display"
+const videoEndpoint = "https://reachms.bfmio.com/bid.json?exchange_id"
```

Comment:
@muncha Looks like you guys have different endpoints that you'd wanna use based on the format. We're working on our end on add a more configurable way to be able to do this. That will let you remove these constants and rather let the Prebid Server host specify them as part of the configuration. We're planning to have that work done by either end of this week or early next week. Would you mind waiting till then and updating this PR after that has been merged in or do you need this to go out sooner than later?

Reply:
Sure, hold off on this. I've got a branch with improvements to this any way. I'll just add the endpoint changes, merge that branch and update this PR. 

Can you point me to an issue or branch to watch for those changes?

Reply:
Is this related to #998?

Reply:
Not that one. It's related to https://github.com/prebid/prebid-server/pull/1070.

Reply:
Cool. That is defiantly on the table for a later update.